### DISCUSSION TITLE
Installazione Home Assistant su portatile HP con ubuntu

### DISCUSSION THREAD

**Discussion by paolodecaro** [indomus/forum#2764] [HUB: Home Assistant]
Buonasera, vi chiedo un piccolo aiuto, Voglio iniziare avventura Indomus. Ho un laptop hp con 4 gb di ram. Ho pensato di instalallare il sistema operativo LINUX Ubuntu. Vi chiedo se è possibile poi installare Indomus e far funzionare il tutto da questo laptop in alternativa a Raspberry.  Grazie in anticipo a chi mi aiuterà.

**Comment by snakuzzo** [comment]
COOOOOOOOSAAAAAAAA?

**Reply by paolodecaro** [reply]
Perdonami sankuzzo,  ho sbagliato qualcosa a porre il mio quesito ?

**Comment by KC-inDomus** [comment]
> Buonasera, vi chiedo un piccolo aiuto, Voglio iniziare avventura Indomus. Ho un laptop hp con 4 gb di ram. Ho pensato di instalallare il sistema operativo LINUX Ubuntu. Vi chiedo se è possibile poi installare Indomus e far funzionare il tutto da questo laptop in alternativa a Raspberry.  Grazie in anticipo a chi mi aiuterà.

ma inDomus è solo il nostro sito... che intendi per installarlo sul portatile

**Reply by paolodecaro** [reply]
Perdonami volevo dire HomeAssist
